server/routes/message.js

```diff
@@ -0,0 +1,46 @@
+var express = require("express");
```

Comment:
Sending a message logs me out. I think it's because the page is being refreshed so there's probably a `preventDefault` you're missing somewhere. Also, the user should not be logged out when the page is refreshed

Reply:
Although I did not implement the message sending, I have fixed this bug.